en/peripherals/mavlink_peripherals.md

```diff
@@ -39,13 +53,16 @@ The `TELEM 1` port is almost always used for the GCS telemetry stream.
 To support this there is a [default serial port mapping](../peripherals/serial_configuration.md#default_port_mapping) of MAVLink instance 0 as shown below:
 - [MAV_0_CONFIG](../advanced_config/parameter_reference.md#MAV_0_CONFIG) = `TELEM 1`
 - [MAV_0_MODE](../advanced_config/parameter_reference.md#MAV_0_MODE) = `Normal`
-- [MAV_0_RATE](../advanced_config/parameter_reference.md#MAV_0_RATE)= `57600` baud
+- [MAV_0_RATE](../advanced_config/parameter_reference.md#MAV_0_RATE)= `1200` B/s
 - [MAV_0_FORWARD](../advanced_config/parameter_reference.md#MAV_0_FORWARD) = `True`
+- [SER_TEL1_BAUD](../advanced_config/parameter_reference.md#SER_TEL1_BAUD) = `57600`
+
 
 ## Example
 
-For example, to use a Companion Computer on `TELEM 2` you might set parameters as shown:
+For example, to use a companion computer on `TELEM 2` you might set parameters as shown:
 - [MAV_2_CONFIG](../advanced_config/parameter_reference.md#MAV_2_CONFIG) = `TELEM 2`
 - [MAV_2_MODE](../advanced_config/parameter_reference.md#MAV_2_MODE) = `Onboard`
-- [MAV_2_RATE](../advanced_config/parameter_reference.md#MAV_2_RATE)= `921600` baud
+- [MAV_2_RATE](../advanced_config/parameter_reference.md#MAV_2_RATE)= `921600` B/s

```

Comment:
What should this be? I know that theoretically the baud and data rate can differ.

Reply:
This depends on the setup (FTDI chip, companion, etc). The simple answer is: I'd use 80'000, which should be enough for most use-cases, but you might have to reduce it if you experience message losses.